libs/sdk-ui-dashboard/src/model/commandHandlers/drill/common/filterValuesResolver.ts

```diff
@@ -54,14 +57,9 @@ export async function resolveFilterValues(
                 invariant(workspace, `workspace needs to be provided for this type of filter: ${filter}`);
                 return resolveAttributeFilterValues(filter, backend, workspace);
             } else {
-                // TODO handle relative date filter resolution
-                return new Promise<IResolvedDateFilterValue>((resolve) =>
-                    resolve({
-                        from: "01-01-2021",
-                        to: "01-01-2021",
-                        granularity: "GDC.time.date",
-                    }),
-                );
+                invariant(backend, `backend needs to be provided for this type of filter: ${filter}`);
```

Comment:
Small detail: the two invariants are in both branches of the if here, would it make sense to extract them above the if?

Reply:
fixed in https://github.com/gooddata/gooddata-ui-sdk/pull/1678/commits/e24281f36d6727f1b201e49e8861bd7895f4d31d